frontend/test/__support__/utils.ts

```diff
@@ -0,0 +1,4 @@
+export const getNextId = (() => {
```

Comment:
This helper was used in the original PR, but is not present in `release-x.47.x` branch yet.
It is copied from `master`.